st2auth_ldap_backend/ldap_backend.py

```diff
@@ -135,8 +136,11 @@ def _ldap_connect(self):
         Prepare ldap object for binding phase.
         """
         try:
-            connection = ldap.initialize(self._ldap_uri)
+            # set ldap options before connecting LDAP server
             ldap.set_option(ldap.OPT_PROTOCOL_VERSION, 3)
+            ldap.set_option(ldap.OPT_REFERRALS, ldap.OPT_OFF)
```

Comment:
Looks like this is going to be conflict w/ https://github.com/StackStorm/st2-auth-backend-ldap/pull/8.